draft-ietf-dprive-dnsoquic.md

```diff
@@ -302,9 +302,15 @@ This means that the client's first query occurs on QUIC stream 0, the second on
 4, and so on (see {{Section 2.1 of RFC9000}}.
 
 Servers MAY defer processing of a query until the STREAM FIN has been indicated
-on the stream selected by the client. Servers and clients MAY monitor the number
-of "dangling" streams for which the expected queries or responses have been received but
-not the STREAM FIN. Implementations MAY impose a limit on the number of
+on the stream selected by the client.
+
+Servers and clients MAY monitor the number
+of "dangling" streams. These are open streams for which
+
+* the expected queries or responses have not been received or,
+* the expected queries or responses have been received but not the STREAM FIN
+
+after implementation defined timeouts. Implementations MAY impose a limit on the number of
 such dangling streams. If limits are encountered, implementations MAY close the connection.
 
```

Comment:
I am not too keen on having the qualifier "after implementation defined timeouts" arriving at the beginning of the paragraph following the two bullets.

Reply:
Yeah - I couldn't get this quite right.... Any better now?

Reply:
Yes, looks good to me.